transactor/src/main/scala/io/mediachain/types/DatastoreTypes.scala

```diff
@@ -0,0 +1,202 @@
+package io.mediachain.types
+
+
+object DatastoreTypes {
```

Comment:
can't we call this just Datastore and put the Datastore interface together as well?
DatastoreTypes is a very ugly name, and it is also rather sad to have a forlorn Datastore trait with one method in its own file.